### PR TITLE
Fork

### PR DESCRIPTION
**Pull request by adrian-enspired**

### CODE REVIEW

File: src/Validator.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    );
+  }
+
+  /**
+   * Validates a Salesforce 18-character alphanumeric Id.
+   *
+   * @param string $value
+   * @throws ValidationException BAD_ID if validation fails
+   */
+  public static function Id(string $value) : void {
```
Comment (robertology) [inline src/Validator.php:91]: capital `I` to conform to some convention in this lib, or was that not desired?

**Review by robertology — CHANGES_REQUESTED** [review]

File: src/Validator.php (hunk 1 above)
Comment (adrian-enspired) [reply src/Validator.php:91]: Salesforce capitalizes all their property names.

**Review by robertology — APPROVED** [review]
Looks good as far as I know :)

**Comment by adrian-enspired** [comment]
one last test to verify we can install fork or prefork version from the same repo (tested on my fork)
```
composer require "nexcess/php-salesforce-rest-api:^51"
composer require "bjsmasth/php-salesforce-rest-api:dev-master#423a84b"
```
both work as expected!

**Pull request merged**